database/migration/migrateData.ts

```diff
@@ -850,6 +851,75 @@ const migrateData = async () => {
     }
   }
 
+  // v0.8.0
+  if (databaseVersionLessThan('0.8.0')) {
+    console.log('Migrating to v0.8.0...')
+
+    console.log(' - Creating permission_flattened view')
+    await DB.changeSchema(
+      `CREATE INDEX IF NOT EXISTS "i_review_status_history_status" ON review_status_history (status);`
+    )
```

Comment:
Can this one go in `44_index.sql` please -- probably next to this line:
```
CREATE INDEX IF NOT EXISTS "i_review_status_history_review_id_fkey" ON review_status_history (review_id);
```

Just keeps all the same things organised in one place.